forte/orbital-helpers/aosubspace.h

```diff
@@ -137,48 +142,29 @@ class AOSubspace {
     // Constructor with list of subspaces
```

Comment:
Please remove the first constructor and give the argument atom_normals an empty map. It's easy to specify this in pybind11, there are other example in the various api files.

Reply:
Let me do that in initialization list.

Reply:
No, I don't think we need to expose this. With one constructor, we would probably will just have to modify the interface of make_aosubspace_projector to deal with the optional variable.